hca/staging_area_validator.py

```diff
@@ -193,11 +194,10 @@ def validate_metadata_file(self, blob: gcs.Blob) -> None:
             assert metadata_id == provenance['document_id']
         if metadata_file := self.metadata_files.get(metadata_id):
             metadata_file['name'] = blob.name
```

Comment:
```suggestion
            metadata_file['name'].add(blob.name)
```
If we are allowing multiple versions of a single metadata file, the value of `metadata_file['name']` should be a `set` instead of a single string.

Reply:
👍 and the field should be defaulted to `set()` as well, will make the change.